### ISSUE TITLE
Code doesn't run :-(

### ISSUE THREAD

**Issue by Ulawun** [llSourcell/Financial_Forecasting_with_TensorflowJS#2]
Hello,
Thx for sharing everything  :-)
I can install without problems.
Clicking on 'Logic' ends with success.
When I click on 'AAPL',
I get the attached error.
How can I get the code to run ?
Thanks.

![llsourcell](https://user-images.githubusercontent.com/5527990/45758973-2c6eb780-bc27-11e8-9d2a-ee66861d98fa.png)

**Comment by Thoughtscript** [comment]
Hi All!

I'm @Thoughtscript - I wanted to apologize about the confusion this has caused! Yikes!

First, let me be very clear that I'm not in any way affiliated with @llSourcell.

This code was NOT PRODUCTION WORTHY and INCOMPLETE - as noted on the original repo listed here - it's a WORK IN PROGRESS.

Actually, I'm not sure how it ended up in a video or attracting any interest whatsoever.

The code was being completed for an article to be posted on the top-ranked consulting firm X-Team's blog. Unfortunately, the company decided to move in another direction with its writing and I've since moved over to Microsoft (though I remain affiliated with X-Team's great community).

Every other repo listed under my @Thoughtscript account without a WIP flag should be fully functional (with any issues, if they exist, noted). This particular project (which you'll note bears my name on in the pictures), is actually no longer maintained.

Any questions, feel free to send me an email at adam.gerard@gmail.com. Sorry for any confusion this has caused.

Cheers!

**Comment by Thoughtscript** [comment]
PS - maybe try:

1. Normalizing the values
1. Correcting the shapes
1. Correctly configuring the convolutional neural net
1. Leveraging parallel processing and WebGL since that shit's gonna break on single-threaded JS

**Comment by gognjanovski** [comment]
I tried to use the code but it didn't worked for me. So the closest working solution that I got was this one: https://towardsdatascience.com/stock-price-prediction-system-using-1d-cnn-with-tensorflow-js-machine-learning-easy-and-fun-fe5323e68ffb 

Hope that helps.
